WordPress/Classes/Models/PublicizeConnection.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import CoreData
+
+@objc public class PublicizeConnection : NSManagedObject
```

Comment:
(No need to change anything!) i _think_ since you're inheriting from NSObject, there's no real need for the `@objc`.